### PR TITLE
Bugs are getting more gross

### PR DESCRIPTION
**Pull request by I-am-Erk**
#### Summary
```SUMMARY: Content "Make bugs much more mutated"```

#### Purpose of change
In order to survive, giant bugs can't just be giant bugs, they'd die pretty quickly.

Luckily, the thing that made them giant is the same thing that can make you, the player, turn into a horrifying freak of nature.

#### Describe the solution
Make bugs into horrifying freaks of nature.

Adds a harvest message to all bugs, and several new butchery items. Currently these items mostly exist to be gross and add lore, but potentially could have some scientific use as well as being absolutely disgusting comestibles.

#### Describe alternatives you've considered
I would like to be able to make snippets describing the bug harvest, instead of a single message per type.

#### Testing
image 
Mostly working.
image
image
Butchery snippets are randomly generated from a series of phrases.

#### Additional context
I plan to add many descriptive snippets for bug organs to suggest variation and mutation.

### CODE REVIEW

**Comment by Zireael07** [comment]
What's garden house doing here?

**Comment by Inglonias** [comment]
Mission accomplished I guess, though I'm slightly worried about the length of the messages.

**Comment by I-am-Erk** [comment]
Nothing, just a bad commit in my repo that I haven't wiped from the record yet.

**Comment by JeanLucVanDamme** [comment]
I love/am disgusted by the new descriptions.

Makes me think of The Fly.

**Comment by kevingranade** [comment]
> Makes me think of The Fly.

So do we want to call them Cronenbugs, or Croninsects?

**Comment by JeanLucVanDamme** [comment]
Cronenbugs is _fantastic_.

There should be a big bug boss, named... David Cronenbug.

**Comment by Xenomorph-III** [comment]
What happened to the item definitions?

**Comment by I-am-Erk** [comment]
PR isn't done.

File: data/json/items/comestibles/carnivore.json [hunk 1]
```diff
@@ -282,10 +285,11 @@
...
     "id": "mutant_bug_organs",
     "type": "COMESTIBLE",
-    "copy-from": "lung",
-    "name": "mutant offal",
-    "name_plural": "mutant offal",
-    "description": "These organs came from a giant mutant bug, and you really aren't sure what to make of them.  There are things you've never seen in any anatomy book, with spines and hair and other unidentified parts protruding off seemingly at random.",
+    "copy-from": "offal",
+    "name": "mutant organs",
+    "name_plural": "mutant organs",
+	"snippet_category": "bug_organs_desc"
```
Comment (anothersimulacrum) [inline data/json/items/comestibles/carnivore.json]:
```suggestion
	"snippet_category": "bug_organs_desc".
```

**Comment by I-am-Erk** [comment]
Still needs testing, but that's enough writing for me for now.

**Comment by I-am-Erk** [comment]
Okay, seems to work. I look forward to hearing what sort of things people do with the new bug parts, I've left them for decoration for the moment.

File: data/json/items/generic.json [hunk 2]
```diff
@@ -318,6 +319,39 @@
...
     "to_hit": -2
   },
+  {
+    "type": "GENERIC",
+    "id": "endochitin",
+    "category": "spare_parts",
+    "symbol": ",",
+    "color": "light_gray",
+    "name": "strand of endochitin",
+    "name_plural": "strands of endochitin",
```
Comment (Night-Pryanik) [inline data/json/items/generic.json]: Replace these with modern `name` object with `str` and `str_pl` fields?

File: data/json/snippets/mutant_anatomy.json [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+      }
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "chitin_desc",
+    "text": [
+      {
+        "id": "chitin_1",
+        "text": "A piece of a bug's exoskeleton, but mutated.  The inner side is lined with veins and strange hooked protruberances."
```
Comment (Night-Pryanik) [inline data/json/snippets/mutant_anatomy.json]:
```suggestion
        "text": "A piece of a bug's exoskeleton, but mutated.  The inner side is lined with veins and strange hooked protuberances."
```

File: data/json/snippets/mutant_anatomy.json [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<bug_harvest_general_chitin>",
+    "text": [
+      "As you peel away the outer shell, you find veins lining the chitin plates",
+      "With the creature dead, its carapace comes away surprisingly easily",
+      "There's a thin membrane, much like skin, spread over the chitin of this creature",
+      "Under the carapace of this mutant is a bristly, velcro like material",
```
Comment (Night-Pryanik) [inline data/json/snippets/mutant_anatomy.json]:
```suggestion
      "Under the carapace of this mutant is a bristly, velcro-like material",
```

File: data/json/snippets/mutant_anatomy.json [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+  {
+    "type": "snippet",
+    "category": "<bug_harvest_general_deeper>",
+    "text": [
+      "Inside, half-formed organs press against spongy meat that doesn't look anything like raw arthropod meat from normal creatures",
+      "You find a collection of hooked spines beneath that seem to have been clasping it on somehow",
+      "Inside is a complex, still-squirming mess of strange appendages and organs that bear only a passing resemblance to any natural creature",
+      "Beneath the chitin, the meat is covered in thick, bristly hair hiding a chaotic bramble of half-formed, mutated organs",
+      "Inside is a tangled mess of organs and tissues that do not appear to entirely natural",
+      "Inside the creature you find lungs, hearts, and intestings more like a mammal than a giant bug"
```
Comment (Night-Pryanik) [inline data/json/snippets/mutant_anatomy.json]:
```suggestion
      "Inside the creature you find lungs, hearts, and intestines more like a mammal than a giant bug"
```

File: data/json/snippets/mutant_anatomy.json [hunk 6]
```diff
@@ -0,0 +1,201 @@
...
+      "Thick, ropey cords of tissue beneath its chitin protect an inner layer of strange organs, resembling those of a bird more than anything"
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "<bug_harvest_acid_descriptive>",
+    "text": [
+      "<bug_harvest_general_descriptive>",
+      "The powerfully acidic vapors coming from the carcass make it hard to work",
+      "The tissues of the creature are full of half-formed organs, their purpose unclear.",
```
Comment (Night-Pryanik) [inline data/json/snippets/mutant_anatomy.json]:
```suggestion
      "The tissues of the creature are full of half-formed organs, their purpose unclear",
```

**Comment by ZhilkinSerg** [comment]
I will close this one to clean up PR queue. Feel free to reopen.

**Pull request closed without merging**